modules/beopBidAdapter.js

```diff
@@ -40,19 +40,33 @@ export const spec = {
     const pageUrl = getPageUrl(bidderRequest.refererInfo, window);
     const gdpr = bidderRequest.gdprConsent;
     const firstSlot = slots[0];
+    const kwdsFromRequest = bidderRequest.ortb2?.site?.keywords;
+    let keywords = [];
+    if (kwdsFromRequest) {
+      if (isArray(kwdsFromRequest)) {
+        keywords = kwdsFromRequest;
+      } else {
```

Comment:
I guess for this whole block 48-54 (both the if and the else), you could paranoia-check `isStr(kwdsFromRequest)` so that you're not pushing functions or numbers to `keywords`…